pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -1466,13 +1484,16 @@ class Build {
                                     repoHandler.setUserDefaultsJson(context, DEFAULTS_JSON)
                                     repoHandler.checkoutUserPipelines(context)
                                 }
+                                printGitRepoInfo()
                             } else {
                                 context.println "[CHECKOUT] Checking out to the user's temurin-build..."
                                 repoHandler.setUserDefaultsJson(context, DEFAULTS_JSON)
                                 repoHandler.checkoutUserBuild(context)
+                                printGitRepoInfo()
```

Comment:
do we want all these call to printGitRepoInfo() or most of them are for debug purpose?

Reply:
So yes, I think it is useful as it shows clearly what you've built with, and especially if there are "Untracked" files left around. It's only a few lines. The trouble we've had in the past in not clearly knowing for sure we've built with the right scripts...